platform/platform-configuration-listener/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+
+<blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <!--ext:property-placeholder/-->
+
+    <ext:property-placeholder>
+        <ext:default-properties>
+            <ext:property name="configFileExtension" value=".config"/>
+            <ext:property name="configFileDirectory" value="${ddf.home}/etc"/>
+        </ext:default-properties>
+    </ext:property-placeholder>
+
+    <reference id="configurationAdmin" interface="org.osgi.service.cm.ConfigurationAdmin"/>
+
+    <bean id="watchThread" class="java.util.concurrent.Executors"
+          factory-method="newSingleThreadExecutor">
+    </bean>
+
+    <bean id="defaultFileSystem" class="java.nio.file.FileSystems" factory-method="getDefault">
+    </bean>
+
+    <bean id="watchService" factory-ref="defaultFileSystem" factory-method="newWatchService">
+    </bean>
+
+    <bean id="persistenceStrategy"
+          class="org.codice.ddf.configuration.store.FelixPersistenceStrategy">
```

Comment:
Is it possible to support multiple persistence strategies?  Do you just have to create multiple file handlers?  I am wondering if it would make sense to have them be OSGi services and they would be injected as a reference list instead.